SecretAgent/SecretAgent/Screen/Board/View/BoardViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  BoardViewController.swift
+//  SecretAgent
+//
+//  Created by Minkyeong Ko on 2022/11/23.
+//
+
+import UIKit
+
+enum BoardSize {
```

Comment:
p3;
하드 코딩을 줄이기 위해 위쪽에 상수값들을 enum을 활용해서 넣어두시는 거 좋네여 ㅎㅎ. 다만  BoardSize가 이 파일에서만 쓰이는 거라면 
```suggestion
private enum BoardSize {
```
은 어떨까요?!

Reply:
사실 BoardSize 안의 badgeWidth와 badgeHeight가 CustomCollectionViewCell 에서도 쓰이긴 하는데, BoardSize가 BoardViewController 파일 안에 있는 게 마음에 걸리네요..
우선은 CustomCollectionViewCell 에 private BadgeSize 이넘을 생성하고 둘 다 private으로 유지하도록 하겠습니다!